ports/cannonball-lr/Cannonball.sh

```diff
@@ -13,21 +13,24 @@ else
 fi
 
 source $controlfolder/control.txt
+source $controlfolder/device_info.txt
 
 get_controls
 
-whichos=$(grep "title=" "/usr/share/plymouth/themes/text.plymouth")
-if [[ $whichos == *"TheRA"* ]]; then
+if [[ $CFW_NAME == "TheRA" ]]; then
   raloc="/opt/retroarch/bin"
   raconf=""
-elif [[ $whichos == *"RetroOZ"* ]]; then
+elif [[ $CFW_NAME == "RetroOZ" ]]; then
   raloc="/opt/retroarch/bin"
   raconf="--config /home/odroid/.config/retroarch/retroarch.cfg"
-elif [[ -e "/storage/.config/.OS_ARCH" ]] || [ -z $ESUDO ]; then
-  raloc="/usr/bin"
+elif [[ $CFW_NAME == "ArkOS" ]]; then
```

Comment:
```suggestion
elif [[ $CFW_NAME == *"ArkOS"* ]]; then
```
or
```suggestion
elif [[ $CFW_NAME == "ArkOS" ]] || [[ $CFW_NAME == "ArkOS wuMMLe" ]]; then
```
Otherwise these won't detect the wuMMLe fork for ArkOS on the 351P/M

Reply:
Sounds good! Should I add 2048 as well?

Reply:
Shoot!  Go for it!